ansible/roles/vm_set/tasks/ptf_portchannel.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Include variables for PTF containers
+  include_vars:
+    dir: "{{ playbook_dir }}/group_vars/ptf_host/"
+
+- name: Set ptf host
+  set_fact:
+    ptf_host: "ptf_{{ vm_set_name }}"
+    ptf_host_ip: "{{ ptf_ip.split('/')[0] }}"
+
+- name: Add ptf host
+  add_host:
+    hostname: "{{ ptf_host }}"
+    ansible_user: "{{ ptf_host_user }}"
+    ansible_ssh_host: "{{ ptf_host_ip }}"
+    ansible_ssh_pass: "{{ ptf_host_pass }}"
+    ansible_python_interpreter: "/usr/bin/python"
```

Comment:
>ansible_python_interpreter: "/usr/bin/python" [](start = 4, length = 45)

Why you need explicitly specifiy python? Possible to remove?

Reply:
https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/vm_set/tasks/announce_routes.yml#L11-L19

https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/vm_set/tasks/start_ptf_tgen.yml#L12-L19

I just follow some other modules to manipulate the ptf container.